webrtc/simplecall.https.html

```diff
@@ -17,7 +17,7 @@
   <script src="/resources/testharness.js"></script>
   <script src="/resources/testharnessreport.js"></script>
   <script type="text/javascript">
-  var test = async_test('Can set up a basic WebRTC call.', {timeout: 5000});
+  var test = async_test('Can set up a basic WebRTC call.');
```

Comment:
Taskcluster is failing on Firefox because this test crashed 1/10 times. That's hardly because of these changes though.